src/odemis/model/_metadata.py

```diff
@@ -107,6 +107,8 @@
 BAND_PASS_THROUGH = "pass-through"  # Special "filter" name when there is no filter: all light passes
 
 MD_AR_POLE = "Angular resolved pole position"  # px, px (tuple of float), position of pole (aka hole center) in raw acquisition of SPARC AR
+MD_AR_MIRROR_TOP = "Top line of the mirror"  # px, px/m (tuple of floats), position of the top of the mirror dependent of the wavelength.
```

Comment:
I think this should be "position of the top of the mirror dependent **on** the wavelength."